core/server/common/src/main/java/alluxio/master/MasterProcess.java

```diff
@@ -159,23 +156,12 @@ void registerServices(GrpcServerBuilder serverBuilder,
   }
 
   @Override
-  public boolean waitForReady(int timeoutMs) {
-    try {
-      CommonUtils.waitFor(this + " to start",
-          () -> {
-            boolean ready = isServing();
-            if (ready && !ServerConfiguration.getBoolean(PropertyKey.TEST_MODE)) {
-              ready &= mWebServer != null && mWebServer.getServer().isRunning();
-            }
-            return ready;
-          }, WaitForOptions.defaults().setTimeoutMs(timeoutMs));
-      return true;
-    } catch (InterruptedException e) {
-      Thread.currentThread().interrupt();
-      return false;
-    } catch (TimeoutException e) {
-      return false;
+  public boolean isReady() {
+    boolean ready = isServing();
```

Comment:
To be more self-explanatory, rename `isServing()` to `isGrpcServing()` and add another `isWebServing()` for line 161-163?